### PR TITLE
Implementing categories and places on the footer

### LINKED ISSUE #1303: Links on the footer don't work / Implementing Categories and locations
**Describe the bug**
Links on the footer don't currently work

**To Reproduce**
1. Go to main page
2. Scroll down 
3. Links don't work

**Expected behavior**

They should work

### PR DESCRIPTION
**Pull request by Kreijstal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
You couldn't navigate with the footer links before

#### Changes proposed in this pull request:

-This adds links to the footer so that you can navigate

Fixes #1303

### CODE REVIEW

**Comment by Kreijstal** [comment]
Fixed issues

File: app/components/footer-main.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 
 export default Component.extend({
-  tagName    : 'footer',
-  classNames : ['ui', 'inverted', 'vertical', 'footer', 'segment'],
-
-  currentLocale: computed(function() {
+  tagName       : 'footer',
+  classNames    : ['ui', 'inverted', 'vertical', 'footer', 'segment'],
+  events        : ['Appearance or Signing', 'Attraction', 'Camp, Trip, or Retreat', 'Class, Training, or Workshop', 'Concert or Performance', 'Conference', 'Convention', 'Dinner or Gala', 'Festival or Fair', 'Game or Competition'],
```
Comment (niranjan94) [inline app/components/footer-main.js]: @Kreijstal this is not what I meant. These should be coming in from the server. FE does not control any of this.

Comment (Kreijstal) [reply app/components/footer-main.js]: Are these already defined from the server?

**Comment by Kreijstal** [comment]
I used event-types for the server API, for places, can't we make a separate issue for this?

**Comment by CosmicCoder96** [comment]
@Kreijstal yeah sure.

**Comment by Kreijstal** [comment]
@CosmicCoder96 @niranjan94 review again

File: app/components/footer-main.js [hunk 2]
```diff
@@ -2,13 +2,16 @@ import Component from '@ember/component';
...
+  tagName       : 'footer',
+  classNames    : ['ui', 'inverted', 'vertical', 'footer', 'segment'],
+  places        : ['India', 'Singapore', 'Berlin', 'New York', 'Hong Kong'],
+  currentLocale : computed(function() {
     return this.get('l10n').getLocale();
   }),
-
+  init() {
+    this._super(...arguments);
+    this.set('events', this.store.findAll('event-type'));
```
[the remark this answers is not in the document]
Comment (Kreijstal) [reply app/components/footer-main.js]: How do you do that?

[the remark this answers is not in the document]
Comment (CosmicCoder96) [reply app/components/footer-main.js]: @Kreijstal  you can include it in the application route model, just like notifications, and then while rendering the component of footer, you can pass it just like notifications are passed to the notification component.

**Pull request merged**